internal/kube/site/site.go

```
@@ -1737,19 +1772,72 @@ func (s *Site) checkSecuredAccess() error {
+				err = fmt.Errorf("RouterAccess %q mixes static and dynamic ports", name)
+				s.logger.Error("RouterAccess mixes static and dynamic ports", slog.String("name", name))
+			} else {
+				err = fmt.Errorf("RouterAccess %q conflicts with %q on port %d", name, withName, withPort)
+				s.logger.Error("RouterAccess port conflicts",
+					slog.String("name", name),
+					slog.String("other", withName),
+					slog.Int("port", withPort),
+				)
+			}
+			la.SetConfigured(err)
+			s.updateRouterAccessStatus(la)
```

golangci-lint (2.12.2)

[error] Error return value of `s.updateRouterAccessStatus` is not checked

(errcheck)